lib/board.rb

```diff
@@ -12,8 +12,35 @@ def self.empty_board
                7, 8, 9])
   end
 
+  def self.with_moves(cells)
+    Board.new(cells)
+  end
+
+  def full?
+    board_positions = (1..9).to_a
+    !board_positions.any? { |free_position| 
```

Comment:
This is a little hard to read for me. 

could you use the `all?` method here instead of `any?`, so then you probably wouldn't need to inverse the result?

Also, using `{ }` is conventially used for single line statements. I feel this is short enough for one line,  although if you do need anything longer be sure to use `do` and `end` instead.

I would advise you try and learn the ruby enumerable library as there's a load of really cool functionality there, and you should be able to make this much more succinct.